src/state/application/atoms.ts

```diff
@@ -0,0 +1,14 @@
+import { atomWithStorage, createJSONStorage } from 'jotai/utils'
+
+/*
+  note: 
+  We should consider a generic sessionStorage abstraction if this pattern becomes common. (i.e., for future promo dismissals like the tax service discounts or FoR launch notification)
```

Comment:
nit: please make sure this is uppercase where necessary (new sentence or heading) and has periods at the endow each sentence.

There are also a few minor issues, such as period after "common" and "FoR" (prefer explicit name).